spec/features/form/check_answers_page_spec.rb

```diff
@@ -127,6 +127,18 @@
       end
     end
 
+    context "when the user changes their answer" do
```

Comment:
Can we make clear that they're changing their answer "from the check answers page" not just in general (for example by clicking back button)?